region/norfair/west/Crumble Shaft.json

```diff
@@ -214,7 +214,14 @@
       "link": [1, 2],
       "name": "Base",
       "requires": [
-        {"heatFrames": 200}
+        {"simpleHeatFrames": 150},
+        {"or": [
+          {"and": [
+            "canMoonfall",
```

Comment:
- The moonfall is hard to pull off most of the time because of the crumble block, so not quite a basic moonfall.
- Landing on the doorframe while moonfalling takes some moonfall delaying actions that aren't really part of basic moonfall.

Covered by heat scaling though.